contracts/service-registry/src/contract.rs

```diff
@@ -336,21 +338,23 @@ pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> Result<Binary, ContractErr
 }
 
 pub mod query {
+    use connection_router::state::ChainName;
+
     use crate::state::{AuthorizationState, WORKERS, WORKERS_PER_CHAIN};
 
     use super::*;
 
     pub fn get_active_workers(
         deps: Deps,
         service_name: String,
-        chain_name: String,
+        chain_name: ChainName,
     ) -> Result<Vec<Worker>, ContractError> {
         let service = SERVICES
             .may_load(deps.storage, &service_name)?
             .ok_or(ContractError::ServiceNotFound)?;
 
         let workers = WORKERS_PER_CHAIN
-            .prefix((&service_name, &chain_name))
+            .prefix((&chain_name, &service_name))
```

Comment:
Actually ignore this, I'd rather just switch the order back to how it was.

Reply:
Was there any reason for the switch?

Reply:
This was switched since `WORKERS_PER_CHAIN` now has ChainName first instead of second.

I'm going to keep trying to figure out how to implement `KeyDeserialize` for the `&ChainName` so I can keep it as second, which would make the switch not necessary.

Reply:
I obviously understand that you had to switch cuz the underlying data object switched. I was asking for the reason for this switch from the beginning, and I guess you kinda answered it. Ya please try to avoid the switch.